### DISCUSSION TITLE
Annotation, text tool issues?

### DISCUSSION THREAD

**Discussion by shockerNY** [ggarra13/mrv2#248] [Q&A]
Thanks for a great tool! I always find it useful.

I have a question to post.

I have updated to the latest mrv2-v1.1.8 and the text tool is behaving strangely.
I downloaded the archive and found that it behaves as expected up to "mrv2-v1.1.3", but the same condition is present in "mrv2-v1.1.4" and later versions.

State description. I don't know if it comes across well,

Annotate with text.
Shif+Enter or double-click to confirm.
The text disappears when you do this.
If you click on a different location with the text tool again, the text will appear.

However, the text is not confirmed or the text remains missing.

We cannot determine whether there has been a change in the text tool specifications or whether this is a bug.
Are there any users in a similar situation?
We would be happy to resolve this issue.

Windows 11 Pro 23H2
Intel(R) Core(TM) i9-14900F 

Translated by Deepl

**Comment by ggarra13** [comment]
> We cannot determine whether there has been a change in the text tool specifications or whether this is a bug.
> Are there any users in a similar situation?

It is a bug.   I was able to reproduce it.  Albeit not exactly how you describe it.  In my case, it seems to disappear once I move the mouse outside the view window.

**Comment by ggarra13** [comment]
Here's a full featured beta of v1.1.9 for you to test.  It should fix the text issue:

.exe:
https://mega.nz/file/vXoCyJQT#M66N18exA2Q1sltsRAnW7p2NmbkSDDKqxrSqphaBUP4

.zip:
https://mega.nz/file/eCpkyRLK#LtoGMDCzx8Vq9aardxX3UKj8vPzZWq5JFOOXwrO6ZNY

**Reply by shockerNY** [reply]
Tried mrv2-v1.1.9.
It works as expected!

Thank you for the quick response! Excellent.
